src/application-logger.ts

```diff
@@ -2,9 +2,11 @@ import pino from 'pino';
 
 export const loggerPrefix = '[Eppo SDK]';
 
+const isNode = typeof window === 'undefined';
+
 // Create a Pino logger instance
 export const logger = pino({
   level: process.env.NODE_ENV === 'production' ? 'warn' : 'info',
   // https://getpino.io/#/docs/browser
-  browser: { disabled: true },
+  browser: { disabled: isNode },
```

Comment:
@leoromanovsky @aarsilv It looks like Pino does support client-side logging, so it seems like we just need to detect to see if we're running in the browser or not.

Reply:
not sure yet -- let me do a bit of testing to verify

Reply:
> however at compile-time is where we are seeing the error
Did the client say this? It looks like the error contains `webpack-internal://` in the trace, which could also be a runtime error coming from https://webpack.js.org/configuration/dev-server/

Reply:
I saw you on the customer thread thanks for asking questions - I might have misunderstood where their error was coming from.

Reply:
Oh nice spot!

Will still be good to reproduce and then verify this fixes it